Jenkinsfile

```diff
@@ -680,6 +680,9 @@ def withBeatsEnv(Map args = [:], Closure body) {
       dockerLogin(secret: "${DOCKER_ELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
       dockerLogin(secret: "${DOCKERHUB_SECRET}", registry: 'docker.io')
     }
+    if (nodeOS().contains('darwin')) {
+      sh 'env | sort'
+    }
```

Comment:
Do you need it on pro?
```suggestion
```

Reply:
No, but somehow https://github.com/elastic/beats/pull/29371 works literally fine, but in this PR is not working out of the box... so I need to debug what's the reason that the `gvm` is not in the PATH.